src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -36,12 +42,6 @@ You'll need:
     - All nodes in each cluster will need access to the Certificate Authority for the other cluster. Refer to [Step 2. Connect from the destination to the source](#step-2-connect-from-the-destination-to-the-source).
 - LDR replicates at the table level, which means clusters can contain other tables that are not part of the LDR job. If both clusters are empty, create the tables that you need to replicate with **identical** schema definitions (excluding indexes) on both clusters. If one cluster already has an existing table that you'll replicate, ensure the other cluster's table definition matches. For more details on the supported schemas, refer to [Schema Validation](#schema-validation).
 
-{% comment  %}To add later, after further dev work{{site.data.alerts.callout_info}}
-If you need to run LDR through a load balancer, use the load balancer IP address as the SQL advertise address on each cluster. It is important to note that using a load balancer with LDR can impair performance.
-{{site.data.alerts.end}}{% endcomment %}
-
-To create bidirectional LDR, you can complete the [optional step](#step-4-optional-set-up-bidirectional-ldr) to start the second LDR job that sends writes from the table on cluster B to the table on cluster A.
-
 ### Schema validation
```

Comment:
What about Schema validation? Is this just the old syntax since the new syntax will create the table?

Reply:
both flavors undergo schema validation. In fact, the `CREATE LOGICALLY REPLICATED` is slightly more restricive. Unlike with the og syntax, a replicating table with the new syntax cannot have:
- UDTs
- foreign keys